manifests/init.pp

```diff
@@ -200,6 +203,7 @@
   Integer[0] $api_service_worker_count = 1,
   Integer[0] $content_service_worker_timeout = 90,
   Integer[0] $api_service_worker_timeout = 90,
+  Optional[Hash] $api_client_auth_cn_map = {},
```

Comment:
There's no need to use `Optional` if there's a default. `Optional[Hash]` is equivalent to `Variant[Hash, Undef]`. You can also enforce stricter types.
```suggestion
  Hash[Stdlib::Fqdn, String[1]] $api_client_auth_cn_map = {},
```

Reply:
I'm not sure what you mean. One is the key, the other the value. Perhaps the value can be even stricter than `String[1]` since it must be a valid Django username, but Django usernames are looser than FQDNs. They could include `@` for example.